controllers/seedimage_controller.go

```diff
@@ -360,7 +360,7 @@ func fillBuildImagePod(name, namespace, baseImg, regURL string) *corev1.Pod {
 						fmt.Sprintf("%s && %s && %s",
 							fmt.Sprintf("curl -Lo base.img %s", baseImg),
 							fmt.Sprintf("curl -ko reg.yaml %s", regURL),
-							"xorriso -indev base.img -outdev /iso/elemental.iso -map reg.yaml /reg.yaml -boot_image any replay"),
+							"xorriso -indev base.img -outdev /iso/elemental.iso -map reg.yaml /livecd-cloud-config.yaml -boot_image any replay"),
```

Comment:
We can also revisit this name if we feel like. I think here and in the cloud-config you found are the only places where this is actually being called.  And the ISO generation script too, but this should be dropped soon IMHO in favor of this feature.

Reply:
It used to be a wildcard (like `*.yaml`) iirc. 🤷🏻‍♂️ 

I wouldn't _change_ the name now, who knows which other people/tools depend on it.

Reply:
It still is a `*.yaml` from `elemental-register` PoV, this filename is just used to decide running `elemental-register` or not. `elemental-register` will read any available yaml file within the given directory.

Anyway, we agreed to leave it as is for now in today's stand up.